src/annotator/plugin/pdf.js

```diff
@@ -94,6 +105,12 @@ export default class PDF {
       return;
     }
 
+    // Handle `PDF` instance being destroyed while URI is fetched. This is only
+    // expected to happen in synchronous tests.
+    if (this._destroyed) {
+      return;
+    }
```

Comment:
This is a little bit of a hack, but it is also simple and easy to change. An alternative in future would be to prevent the check for selectable text from always being run when this class is instantiated.